test/Sign.Core.Test/SignatureProviders/AggregatingSignatureProviderTests.Containers.cs

```diff
@@ -0,0 +1,235 @@
+namespace Sign.Core.Test
```

Comment:
Should this code have the .NET Foundation header?

Reply:
See https://github.com/dotnet/sign/issues/449.  I will address this in a separate PR.